Example/CardParts/CardPartMapViewCardController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  CardPartMapViewCardController.swift
+//  CardParts_Example
+//
+//  Created by Michael VanDyke on 10/5/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import CardParts
+import RxCocoa
+import RxSwift
+import CoreLocation
+import MapKit
+
+class CardPartMapViewCardController: CardPartsViewController {
+    
+    let viewModel = ReactiveCardPartMapViewModel()
+    
+    let cardPartTextView = CardPartTextView(type: .normal)
+    let cardPartTextField = CardPartTextField(format: .none)
+    let cardPartMapView = CardPartMapView(type: .standard, location: CLLocation(latitude: 37.430489, longitude: -122.096260), span: MKCoordinateSpan(latitudeDelta: 1, longitudeDelta: 1))
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        cardPartTextView.text = "This is a CardPartMapView"
+        cardPartTextField.placeholder = "Enter Address"
+ 
+        setupCardParts([
+            cardPartTextView,
+            cardPartTextField,
+            cardPartMapView
+        ])
+        
+        setupConstraints()
+        setupBindings()
+    }
+    
+    private func setupConstraints() {
```

Comment:
I like this, but I'm curious if developers will be confused as to why the map isn't showing if they forget to set a height constraint?  We could put a fixed height in the `CardPartMapView` initializer but I'm not sure I'm sold on that either.  @bharathmurs @badrinathvm any input?

Reply:
Throw an error of some sort if there isn't a height set?

Reply:
Yeah, I flip-flopped on this for a bit before I landed on this implementation.

Reply:
@codethebeard Anything added inside the `CardPart ` will have height set up automatically. If mapView doesn't have intrinsic height can we create a `open var intrinsicHeight { }` and set to default value = 300 or magic number .. and since it's open variable users can set to height of their choice later in their apps.

Reply:
Sounds good, I can do that.